content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -9,6 +9,8 @@ tocDepth: 2
 
 This document describes the Prisma CLI commands, arguments, and options.
 
+Command should be prepended by npx, pnpx, or yarn if prisma is installed locally with npm, pnpm, or yarn respectivly.
```

Comment:
Thanks @ObamaTheLlama114 for adding this note. Let's put it into an <Admonition> and perhaps make it slightly less terse... maybe something like "Be aware that Prisma commands are typically prepended by `npx`, `pnpx`, or `yarn`, if Prisma is installed locally with npm, pnpm, or yarn respectively.

Reply:
fixed